files/register_route53.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this script should be in the AMI. 
Route 53 is something specific for the Terraform deployment.

Reply:
So are the volumes. It doesn't hurt to be here if it means the user data script will be simplified.
Nobody is required to use it, but if someone wants to - they can. Route 53, at least for the moment, is the only viable option for a cluster.